### PR TITLE
Update tests to release CoreCustomAttributes EC3.2 schema

### PR DESCRIPTION
**Pull request by akshay05n**
CoreCustomAttributes v01.00.04, an EC3.2 schema, is about to be released. To ensure that PR build pipelines can work with the latest schema, some tests and test data need to be modified. This PR introduces the required test changes.

itwinjs-core: https://github.com/iTwin/itwinjs-core/pull/6141

### CODE REVIEW

File: schemas/BisSchemas/BisSchemas.PartFile.xml [hunk 1]
```diff
@@ -600,7 +600,7 @@
     </Part>
 
     <Part Name="SchemaUpgradeCustomAttributes" BentleyBuildMakeFile="findSchemaPath.mki" BentleyBuildMakeOptions="-dSYMLINK_OUT=$(BuildContext)ECSchemas/Standard/SchemaUpgradeCustomAttributes.01.00.00.ecschema.xml -dSCHEMA_PATTERN=SchemaUpgradeCustomAttributes.*ecschema.xml"/>
-    <Part Name="CoreCustomAttributes" BentleyBuildMakeFile="findSchemaPath.mki" BentleyBuildMakeOptions="-dSYMLINK_OUT=$(BuildContext)ECSchemas/Standard/CoreCustomAttributes.01.00.03.ecschema.xml -dSCHEMA_PATTERN=CoreCustomAttributes.*ecschema.xml"/>
+    <Part Name="CoreCustomAttributes" BentleyBuildMakeFile="findSchemaPath.mki" BentleyBuildMakeOptions="-dSYMLINK_OUT=$(BuildContext)ECSchemas/Standard/CoreCustomAttributes.01.00.04.ecschema.xml -dSCHEMA_PATTERN=CoreCustomAttributes.*ecschema.xml"/>
```
Comment (naveedkhan8067) [inline schemas/BisSchemas/BisSchemas.PartFile.xml:603]: I think this change will break the `imodel-native` build as this schema is not present at the moment.

Comment (akshay05n) [reply schemas/BisSchemas/BisSchemas.PartFile.xml:603]: The `imodel-native` build will not be affected by this change once the CoreCustomAttributes EC3.2 schema is released. The build pipelines will work fine with the latest schema.

Comment (naveedkhan8067) [reply schemas/BisSchemas/BisSchemas.PartFile.xml:603]: yes, we have to first complete the PR in `bis-schemas` repository. I will double check if the changes in both PR's are compatible with each other.

**Review by ColinKerr — APPROVED** [review]

**Review by rschili — APPROVED** [review]

**Review by swwilson-bsi — APPROVED** [review]

**Review by ben-polinsky — APPROVED** [review]

File: schemas/BisSchemas/BisSchemas.PartFile.xml (hunk 1 above)
Comment (naveedkhan8067) [reply schemas/BisSchemas/BisSchemas.PartFile.xml:603]: `bis-schemas` [PR](https://github.com/iTwin/bis-schemas/pull/246) is now complete and the CoreCustomAttributes schema version `01.00.04` is released so the builds should pass now.

**Review by naveedkhan8067 — APPROVED** [review]

**Pull request merged**